scripts/swarm-fix-route.sh

```diff
@@ -380,6 +380,12 @@ function get_overlay_net_prefix {
   fi
 }
 
+function flushNetfilterRules {
+  echo "flushing connection with conntrack... "
+  sudo conntrack -F
+  exit 1
```

Comment:
Why exit 1 ?
That will exit without executing the rest of the script.
More over in shell when a script went well it exit with 0.

Reply:
Correct, my bad, let me change this